test/library/run-options.test.js

```diff
@@ -239,4 +241,29 @@ describe('Newman run options', function () {
             });
         });
     });
+
+    describe('Option --ignore-iteration ', function () {
+        var iterationProperty = 'run.stats.iterations.total',
+            collectionRunPath = path.join(__dirname, '..', '..', 'out', 'iteration-count-test.json');
```

Comment:
You can simplify this:
```suggestion
            collectionRunPath = path.join(__dirname, '../../out/iteration-count-test.json');
```